### ISSUE TITLE
Module not found: Can't resolve 'fs'

### ISSUE THREAD

**Issue by oste** [finkef/react-native-tailwind.macro#2]
I am noticing the following error. I guess it is trying to use fs on the frontend?

```
error - ../../node_modules/react-native-tailwind.macro/dist/macro/resolve-tailwind-config.js:3:0
Module not found: Can't resolve 'fs'

Import trace for requested module:
../../node_modules/react-native-tailwind.macro/dist/macro/index.js
../../node_modules/react-native-tailwind.macro/dist/index.js
./src/pages/index.tsx
```

**Comment by finkef** [comment]
Looks like the macro didn’t transform the code to reference the proper imports.
Did you install and add `babel-plugin-macros` to your `.babelrc`?

**Comment by oste** [comment]
it seems I was missing macros in my babel config.

**Comment by oste** [comment]
Although I am noticing now that my tailwind config doesn’t seem to be used. The default configs are showing up

**Comment by finkef** [comment]
It looks for the tailwind config in the current and it's parents directories until it finds a package.json file (at which point it assumes that is the project root). You may need to restart the packager after applying changes to the config file, maybe even clear the babel cache. We should add that to the project's caveats.

**Comment by oste** [comment]
I think the cause of my issues is related to using yarn workspaces. So my tailwindconfig lives in one workspace while my component lives in another workspace 😬

**Comment by finkef** [comment]
Oh, gotcha! As quick workaround, you could add a config file that simply re-exports your regular config from the workspace directory.
I guess we should add some configuration options to the macro to specify an alternative path to the config file. I'll take a look at this this weekend. 👍

**Comment by oste** [comment]
I tried to do this in my component directory

```
const Config = require('../../../apps/sports/src/tailwind.config');

module.exports = Config;
```

However I unfortunately have imports in my tailwindconfig and those didn't seem to resolve. So I end up with a new error `Cannot find module './types/colors`. I will probably just localize those for now but also interested in helping out with the config resolution.

**Comment by finkef** [comment]
Just published 1.2.0 which allows you to pass in a custom config path, see [here](https://github.com/finkef/react-native-tailwind.macro#macro-options). This should hopefully solve monorepo setups 🎉

**Comment by oste** [comment]
Nice work! it works for me.

Are you planning to add a license?
